### DISCUSSION TITLE
Choisir son genre

### DISCUSSION THREAD

**Discussion by Akutana** [Crownicles/Crownicles#1697] [Suggestions fermées]
Je sais que c'est secondaire par rapport aux autres propositions liées au fonctionnement du jeu, mais on pourrait choisir (et le modifier plus tard) quel pronom on voudrait pour parler de nous. Parce que là les personnes de genre féminin qui se font appeler "il" à tout bout de champ perso je trouve pas ça ouf...
Voilà, ya pas de problème si vous jugez que c'est useless mais je pense que ça pourrait être un ajout sympa !

**Comment by Nargacaura** [comment]
Ça pourrait effectivement être sympa, mais le problème principal est qu'il faudra revoir quasiment *tous* les JSONs afin de prendre en compte le pronom choisi, ce qui représente autant voire plus de travail que de mettre une description à chaque objet...

**Comment by Pietagorh** [comment]
C'est pas useless j'pense parce que ça tient à coeur à certaines personnes, par contre est-ce que ça vaut *vraiment* les dizaines d'heures de modif de json ? (ça représente bien plus de taff que mettre des descriptions à des objets d'après moi)

**Comment by BastLast** [comment]
hello, quasi sur que cette suggestion a déjà été proposée et ça a juste été fait ainsi pour des raisons techniques,

Prévoir 2 genre = coder 2 fois plus parce que faut prévoir les genre partout et ça devient très vite insuportable (c'est déjà grave relou pour les pets et les accords en francais vs en anglais...

Bref quitte à passer 300h sur draftbot j'ai vraiment envie de les passer à faire autre chose.

Faut vraiment se dire que la charge de travail est ÉNORME.

C'est pas que je suis contre l'idée c'est juste que je ne le ferai pas et que même si j'offrais 500 balles à qqun pr le faire je ne trouverai pas de volontaire. (tellement ya du taf et tellement il est chiant)

**Reply by Pietagorh** [reply]
Après comme d'hab' si y'a quelqu'un qui veut s'en occuper... :)

**Reply by ghosts-arch** [reply]
👀

**Reply by romain22222** [reply]
ALORS JE VAIS RAJOUTER UNE PRECISION :

NON ca prendrait pas 300h de travail pour une raison simple : il suffirait à un endroit spécifique (le formattage) de rajouter une règle qui permettrait de dire masculin / féminin. Cependant, il faudrait reprendre tous les textes qui possèdent du masculin / féminin et de les retravailler afin d'inclure les deux. Ca demande une grosse relecture de TOUS les textes, mais la tâche n'est pas infaisable.

Après, je pense pas que c'est dans la volonté des collaborateurs actuels de vouloir travailler une n-ième fois sur le formattage des textes à l'heure actuelle. Il est cependant possible pour un *nouveau* collaborateur d'essayer d'en faire une branche si celui-ci se sent l'âme de le faire, compte tenu de la tâche à effectuer.

**Reply by BastLast** [reply]
Bro ca va prendre bien plis que 300 h si tu veux mon avis,
Rien que cette discussion a occupé 5 personnes pdt au moins 20 min ça fait déjà 1h40 et on a meme pas commencé à bosser dessus.

Je suis d'accord que 90% du taf serait de la réécriture de texte mais entre cet exercice et la GIGA TONNE  de typo qu'on aurait raté + le temps de les signaler et de les corriger yen a pr énormément de temps. 300h cest environ 1 mois avec 2 personnes qui sont dessus à plein temps. C'est à la fois énorme et à la fois ça satteint vite en bossant sur des trucs comme ça.

En plus de ca faut prendre en compte tout le temps perdu en rédaction dans tous les futurd events et savoir ce qu'il se passe quand ik faut reformuler toute la phrase (parce que des fois c'est pas juste rajouter un e)

Ah et ya pas que la commande report faudrait aussi faire les commandes annexes genre inventaire profile, drink etc chui sur ya des trucs qui se cachent un peu partout et pareil pour ca va etre mega relou a faire.

En bref c'est vraiment super long stop le minimiser.

Je met un gros veto dessus, on a des trucs plus importants à faire.

**Reply by romain22222** [reply]
draft t'as pas 40 vétos ptdr

**Reply by romain22222** [reply]
et je confirme que non ca prendra moins de 300h

**Comment by leuhvraiminoryo** [comment]
sinon tu joues en anglais et y'a pas de problème par rapport à ça. La plupart des mentions de joueurs sont avec des pronoms neutres en cette langue :slightly_smiling_face:

**Reply by BastLast** [reply]
+1

**Reply by Pietagorh** [reply]
++

**Reply by Akutana** [reply]
Oui sauf que la plupart des joueurs jouent en fr X)

**Comment by Akutana** [comment]
Ouais je comprends, ça ferait du temps qui aurait pu être passé sur une mécanique de jeu en plus 😅

**Comment by ghosts-arch** [comment]
je rejoint draft sur le fait que c'est l'enfer à faire la maintenant, c'est une feature qui aurait du etre pensée et rajoutée au début du jeu ... ca touche tellement de parties du code (db, jsons d'events et j'en passe) que ca serait super long à coder ... 

au fond, ca fait 3 ans que le bot utilise exclusivement le masculin et le monde tourne encore donc ca peut durer comme ca encore longtemps

**Reply by Bombafox** [reply]
Après au cas où vous auriez oublié l'histoire du jeu... 👀

On est un prince à la recherche de la princesse donc logique que le pronom soit -il- 😱

Puis sinon faudrais aussi modif tte les phrases ou ça parle de la princesse en prince 😑

Puis si on commence à faire ça on va devoir faire tout les genre sinon ça va faire polémique 😴

Puis perso j'ai la flemme de croiser une fourchette volante de combat a rayons lazers sur mes rapports 🥱

**Reply by Pietagorh** [reply]
Plus boomer tu meurs (ça rime z'avez vu)

**Reply by BastLast** [reply]
mer ça rime pas avec meur

**Reply by Pietagorh** [reply]
\+ pourquoi tu pourrais pas être une meuf qui veut épouser la princesse ?

**Reply by Pietagorh** [reply]
> mer ça rime pas avec meur

Tu dis boumeur ou bo omé ?

**Reply by Pietagorh** [reply]
(+ ratio :kappa:)

**Reply by BastLast** [reply]
boumère

**Reply by Pietagorh** [reply]
berk Drapht

**Reply by Bombafox** [reply]
Après j'ai jamais dit que je voulais pas
Mais vous avez demandé à la princesse son avis ? 🤡

Faudrait genrer la princesse et c'est useless aussi.

Faut arrêter de vouloir rendre le jeux compliqué😦

A la limite changer le pronom -il- par -vous-
Et les phrases 
"vous croisez @user, il vous montre...."
Par
"vous croisez @user qui vous montre" 
(jsp si ça existe déjà le deuxième)

pourquoi pas

ou alors changer les pronoms par les pseudo et le problème est réglé

**Comment by BastLast** [comment]
bon je lock ce thread ça spam mes mails et ça n'apporte rien
